app/presenters/header_presenter.rb

```diff
@@ -0,0 +1,13 @@
+class HeaderPresenter < Rectify::Presenter
+  attribute :current_order
+
+  def all_categories
+    Category.all
+  end
+
+  def count_books_in_cart
+    return 0 if !current_order || current_order.complete?
```

Comment:
I think method `current_order` should not return completed order, so you can:
`current_order&.line_items&.map(&:quantity)&.sum.to_i`